packages/techdocs-common/src/stages/publish/local.ts

```diff
@@ -141,12 +142,20 @@ export class LocalPublish implements PublisherBase {
     return express.static(staticDocsDir);
   }
 
+  async getStorageUrl() {
+    const discoveryApi = SingleHostDiscovery.fromConfig(this.config);
```

Comment:
Just noting that this is not required, since we already have an instance in `this.discovery`.